delfin/task_manager/manager.py

```diff
@@ -39,6 +52,48 @@ def __init__(self, service_name=None, *args, **kwargs):
         self.alert_sync = alerts.AlertSyncTask()
         super(TaskManager, self).__init__(*args, **kwargs)
 
+    def _run_scheduler(self, interval, storage_id, is_history, resource,
+                       tag=''):
+        schedule_obj = StorageController()
+        schedule.every(interval).seconds.do(
+            schedule_obj.perf_collect, storage_id, interval,
+            is_history, resource).tag(tag)
+
+        if tag:
+            schedule.run_pending()
+            schedule.clear(tag)
+
+    def periodic_performance_collect(self):
```

Comment:
How is this function getting invoked?

Reply:
This will be invoked during start of service